plugins/osdn/multitenant/plugins.go

```diff
@@ -52,11 +52,7 @@ func (plugin *MultitenantPlugin) SetUpPod(namespace string, name string, id kube
 }
 
 func (plugin *MultitenantPlugin) TearDownPod(namespace string, name string, id kubeletTypes.DockerID) error {
-	vnid, found := plugin.OvsController.VNIDMap[namespace]
-	if !found {
-		return fmt.Errorf("Error fetching VNID for namespace: %s", namespace)
-	}
-	out, err := utilexec.New().Command(plugin.getExecutable(), tearDownCmd, namespace, name, string(id), strconv.FormatUint(uint64(vnid), 10)).CombinedOutput()
+	out, err := utilexec.New().Command(plugin.getExecutable(), tearDownCmd, namespace, name, string(id), "-1").CombinedOutput()
```

Comment:
We should probably put a comment here about the executable not caring about vnid.